tests/node/test_wakunode_store.nim

```diff
@@ -268,52 +263,55 @@ suite "Waku Store - End to End - Sorted Archive":
 
         # Then the response contains the messages
         check:
-          queryResponse1.get().messages == archiveMessages[0..<2]
+          queryResponse1.get().messages == archiveMessages[0 ..< 2]
 
         # Given the next query (2/3)
         let historyQuery2 = HistoryQuery(
           cursor: queryResponse1.get().cursor,
           pubsubTopic: some(pubsubTopic),
           contentTopics: contentTopicSeq,
           direction: true,
-          pageSize: 4
+          pageSize: 4,
         )
 
         # When making the next history query
         let queryResponse2 = await client.query(historyQuery2, serverRemotePeerInfo)
 
         # Then the response contains the messages
         check:
-          queryResponse2.get().messages == archiveMessages[2..<6]
+          queryResponse2.get().messages == archiveMessages[2 ..< 6]
 
         # Given the next query (3/3)
         let historyQuery3 = HistoryQuery(
           cursor: queryResponse2.get().cursor,
           pubsubTopic: some(pubsubTopic),
           contentTopics: contentTopicSeq,
           direction: true,
-          pageSize: 6
+          pageSize: 6,
         )
 
         # When making the next history query
         let queryResponse3 = await client.query(historyQuery3, serverRemotePeerInfo)
 
         # Then the response contains the messages
         check:
-          queryResponse3.get().messages == archiveMessages[6..<10]
+          queryResponse3.get().messages == archiveMessages[6 ..< 10]
 
       asyncTest "Pagination with Zero Page Size (Behaves as DefaultPageSize)":
         # Given a message list of size higher than the default page size
         let currentStoreLen = uint((await archiveDriver.getMessagesCount()).get())
-        assert archive.DefaultPageSize > currentStoreLen, "This test requires a store with more than (DefaultPageSize) messages"
+        assert archive.DefaultPageSize > currentStoreLen,
+          "This test requires a store with more than (DefaultPageSize) messages"
         let missingMessagesAmount = archive.DefaultPageSize - currentStoreLen + 5
 
         let lastMessageTimestamp = archiveMessages[archiveMessages.len - 1].timestamp
         var extraMessages: seq[WakuMessage] = @[]
-        for i in 0..<missingMessagesAmount:
-          let 
-            timestampOffset = 10 * int(i + 1) # + 1 to avoid collision with existing messages
-            message: WakuMessage = fakeWakuMessage(@[byte i], ts=ts(timestampOffset, lastMessageTimestamp))
+        for i in 0 ..< missingMessagesAmount:
+          let
+            timestampOffset = 10 * int(i + 1)
+              # + 1 to avoid collision with existing messages
```

Comment:
This inline comment should be kept inline too :)

Reply:
when a comment doesn't fit the 88-char limit, we generally move it to a new line and indent it (to highlight that it "belongs" to the previous, less indented item)

Reply:
see the "assert" just a few lines above for a more obvious example  of this same principle